tendermint/src/vote.rs

```diff
@@ -67,6 +71,31 @@ impl Vote {
     }
 }
 
+impl lite::Vote for Vote {
+    fn validator_id(&self) -> account::Id {
+        self.validator_address
+    }
+
+    fn sign_bytes<B>(&self, mut sign_bytes: &mut B)
+    where
+        B: BufMut,
+    {
+        // TODO: 1) everytime we encode sth. an error can occur. Change the trait to return a result
+        // instead to enable proper error handling.
+        // 2) Figure out where the chain_id should come from (if sign_bytes remains on Vote, the
```

Comment:
Good point - may have been thinking we'd use something else here, eg. the CanonicalVote, which contains the ChainID

Reply:
Can we wrap the CanonicalVote in something that also contains the ID and the signature?

Reply:
I think that makes sense. Will create a separate struct tomorrow. Feel free to merge this (if you want), I won't forget this change.

Reply:
Done in c941c95. Name of the new wrapping type `SignedVote`.